### PR TITLE
project init command added

### PR DESCRIPTION
**Pull request by ExtinctWolf83**

### CODE REVIEW

File: deploifai/project/init.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+)
+from deploifai.utilities.config import local_config
+
+
+@click.command()
+@pass_deploifai_context_obj
+@is_authenticated
+def init(context: DeploifaiContextObj):
+    """
+    Store project information on your local device
```
Comment (98sean98) [inline deploifai/project/init.py:21]: should be `Initialize current working directory as a Deploifai project`. I think we're gonna follow the convention we have for dataset init as well, to just initialize the current working directory as the project directory

File: deploifai/project/init.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        project_info = answers["project_option"]
+        context.debug_msg(project_info)
+        project_id = project_info["id"]
+        project_name = project_info["name"]
+    except DeploifaiAPIError as err:
+        click.echo(err)
+        raise Abort()
+
+    try:
+        os.mkdir(project_name)
```
Comment (98sean98) [inline deploifai/project/init.py:75]: Don't need to create a project directory, because the user should already be in it.

Comment (98sean98) [reply deploifai/project/init.py:75]: Probably just need to create a `.deploifai` directory with the local config file. If the local config file is already present, we could throw an error for already initialized.

File: deploifai/project/init.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        click.echo(
+            "Creating a project directory, which stores the local config file inside the .deploifai directory. "
+        )
+    except FileExistsError as err:
+        click.echo("Project directory already exists")
+        return
+
+    try:
+        local_config.set_project_config(project_id, context.local_config)
+    except local_config.DeploifaiDataAlreadyInitialisedError:
```
Comment (98sean98) [inline deploifai/project/init.py:88]: Seems like the wrong exception is being used here.

**Review by 98sean98 — CHANGES_REQUESTED** [review]
Requires functional improvements.

File: deploifai/project/init.py (hunk 1 above)
Comment (ExtinctWolf83) [reply deploifai/project/init.py:21]: okay

File: deploifai/project/init.py (hunk 3 above)
Comment (ExtinctWolf83) [reply deploifai/project/init.py:88]: yes
changed it to the right one

File: deploifai/project/init.py (hunk 2 above)
Comment (ExtinctWolf83) [reply deploifai/project/init.py]: refactored

File: deploifai/project/init.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                workspace=command_workspace, fragment=fragment
+            )
+        if not len(project_info):
+            click.echo("No projects in the workspace")
+            raise click.Abort()
+        questions = [
+            {
+                "type": "list",
+                "name": "project_option",
+                "message": "Choose the project to store on local device.",
```
Comment (98sean98) [inline deploifai/project/init.py:55]: Should be `Choose a project to initialize as`

**Review by 98sean98 — CHANGES_REQUESTED** [review]
Just some small semantic changes, then we're good.

**Review by 98sean98 — APPROVED** [review]

**Pull request merged**